compass/landice/tests/antarctica/mesh.py

```diff
@@ -0,0 +1,218 @@
+        self.add_output_file(filename='Antarctica_8to80km.nc')
+        self.add_input_file(
+            filename='antarctica_8km_2020_10_20.nc',
+            target='antarctica_8km_2020_10_20.nc',
+            database='')
+
+    # no setup() method is needed
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+        config = self.config
+        section = config['antarctica']
+
+        logger.info('calling build_cell_width')
+        cell_width, x1, y1, geom_points, geom_edges, floodFillMask = \
+            self.build_cell_width()
+        logger.info('calling build_planar_mesh')
+        build_planar_mesh(cell_width, x1, y1, geom_points,
+                          geom_edges, logger=logger)
+        dsMesh = xarray.open_dataset('base_mesh.nc')
+        logger.info('culling mesh')
+        dsMesh = cull(dsMesh, logger=logger)
+        logger.info('converting to MPAS mesh')
+        dsMesh = convert(dsMesh, logger=logger)
+        logger.info('writing grid_converted.nc')
+        write_netcdf(dsMesh, 'grid_converted.nc')
+        levels = section.get('levels')
+        logger.info('calling create_landice_grid_from_generic_MPAS_grid.py')
+        args = ['create_landice_grid_from_generic_MPAS_grid.py',
+                '-i', 'grid_converted.nc',
+                '-o', 'ais_8km_preCull.nc',
+                '-l', levels, '-v', 'glimmer']
+        check_call(args, logger=logger)
+
+        # Apply floodFillMask to thickness field to help with culling
+        copyfile('antarctica_8km_2020_10_20.nc',
+                 'antarctica_8km_2020_10_20_floodFillMask.nc')
+        gg = netCDF4.Dataset('antarctica_8km_2020_10_20_floodFillMask.nc',
+                             'r+')
+        gg.variables['thk'][0, :, :] *= floodFillMask
+        gg.variables['vx'][0, :, :] *= floodFillMask
+        gg.variables['vy'][0, :, :] *= floodFillMask
+        gg.close()
+
+        # This step uses a subset of the whole Antarctica dataset trimmed to
+        # the region around Antarctica, to speed up interpolation.
+        # This could also be replaced with the full Antarctic Ice Sheet
+        # dataset.
+        logger.info('calling interpolate_to_mpasli_grid.py')
+        args = ['interpolate_to_mpasli_grid.py', '-s',
+                'antarctica_8km_2020_10_20_floodFillMask.nc', '-d',
+                'ais_8km_preCull.nc', '-m', 'b', '-t']
+
+        check_call(args, logger=logger)
+
+        # This step is only necessary if you wish to cull a certain
+        # distance from the ice margin, within the bounds defined by
+        # the GeoJSON file.
+        cullDistance = section.get('cull_distance')
+        if float(cullDistance) > 0.:
+            logger.info('calling define_cullMask.py')
+            args = ['define_cullMask.py', '-f',
+                    'ais_8km_preCull.nc', '-m'
+                                          'distance', '-d', cullDistance]
```

Comment:
```suggestion
                    'ais_8km_preCull.nc', '-m',
                    'distance', '-d', cullDistance]
```
There was a missing comma here - the culling might not be doing what we think it is doing, because the `-m` is probably being ignored: `Running: define_cullMask.py -f ais_8km_preCull.nc -mdistance -d 70.0`
I only noticed this because this step takes so long, and I was staring at it on my machine for a long time. :)

I'll rerun after making this change locally and confirm.

Reply:
This is a good reason to switch from the `OptionParser` to the `ArgumentParser` as a general rule.  This would be an error in the `ArgumentParser`.

Reply:
That comment is obviously outside the scope of this PR...

Reply:
Oh! Thanks for spotting that @matthewhoffman! Hmm I wonder how all the previous tests were running fine in other machines. 

This also means that the same line of the `mesh.py` code in humboldt test case wants to be fixed: 
https://github.com/MPAS-Dev/compass/blob/master/compass/landice/tests/humboldt/mesh.py#L105.

I'm happy to make a separate PR fixing this, or simply include in the current PR even though this PR is for Antarctic mesh. What do you think?